onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -351,6 +352,9 @@ void addGlobalMethods(py::module& m, const Environment& env) {
   m.def(
       "get_device", []() -> std::string { return BACKEND_DEVICE; },
       "Return the device used to compute the prediction (CPU, MKL, ...)");
+  m.def(
+      "seed", [](const int64_t seed) { utils::SetRandomSeed(static_cast<uint64_t>(seed)); },
```

Comment:
static_cast<uint64_t> not needed, right? Lambda binding is already enforcing it to be int64_t, as the SetRandomSeed signature requires it